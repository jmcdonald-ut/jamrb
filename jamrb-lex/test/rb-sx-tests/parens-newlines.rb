(5 + 5)

x = (5 + 5)

y = (3 + 2) - 4

def sum(x, y)
  x + y
end

z = sum(5, 5)
u = sum 5, 5
