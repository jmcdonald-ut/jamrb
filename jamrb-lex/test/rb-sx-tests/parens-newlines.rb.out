((Token ((1 0) on_lparen "("))
 (Token ((1 1) on_int "5"))
 (Token ((1 2) on_sp " "))
 (Token ((1 3) on_op "+"))
 (Token ((1 4) on_sp " "))
 (Token ((1 5) on_int "5"))
 (Token ((1 6) on_rparen ")"))
 (Token ((1 7) on_nl "\n"))
 (Token ((2 0) on_ignored_nl "\n"))
 (Token ((3 0) on_ident "x"))
 (Token ((3 1) on_sp " "))
 (Token ((3 2) on_op "="))
 (Token ((3 3) on_sp " "))
 (Token ((3 4) on_lparen "("))
 (Token ((3 5) on_int "5"))
 (Token ((3 6) on_sp " "))
 (Token ((3 7) on_op "+"))
 (Token ((3 8) on_sp " "))
 (Token ((3 9) on_int "5"))
 (Token ((3 10) on_rparen ")"))
 (Token ((3 11) on_nl "\n"))
 (Token ((4 0) on_ignored_nl "\n"))
 (Token ((5 0) on_ident "y"))
 (Token ((5 1) on_sp " "))
 (Token ((5 2) on_op "="))
 (Token ((5 3) on_sp " "))
 (Token ((5 4) on_lparen "("))
 (Token ((5 5) on_int "3"))
 (Token ((5 6) on_sp " "))
 (Token ((5 7) on_op "+"))
 (Token ((5 8) on_sp " "))
 (Token ((5 9) on_int "2"))
 (Token ((5 10) on_rparen ")"))
 (Token ((5 11) on_sp " "))
 (Token ((5 12) on_op "-"))
 (Token ((5 13) on_sp " "))
 (Token ((5 14) on_int "4"))
 (Token ((5 15) on_nl "\n"))
 (Token ((6 0) on_ignored_nl "\n"))
 (Token ((7 0) on_kw "def"))
 (Token ((7 3) on_sp " "))
 (Token ((7 4) on_ident "sum"))
 (Token ((7 7) on_lparen "("))
 (Token ((7 8) on_ident "x"))
 (Token ((7 9) on_comma ","))
 (Token ((7 10) on_sp " "))
 (Token ((7 11) on_ident "y"))
 (Token ((7 12) on_rparen ")"))
 (Token ((7 13) on_ignored_nl "\n"))
 (Token ((8 0) on_sp "  "))
 (Token ((8 2) on_ident "x"))
 (Token ((8 3) on_sp " "))
 (Token ((8 4) on_op "+"))
 (Token ((8 5) on_sp " "))
 (Token ((8 6) on_ident "y"))
 (Token ((8 7) on_nl "\n"))
 (Token ((9 0) on_kw "end"))
 (Token ((9 3) on_nl "\n"))
 (Token ((10 0) on_ignored_nl "\n"))
 (Token ((11 0) on_ident "z"))
 (Token ((11 1) on_sp " "))
 (Token ((11 2) on_op "="))
 (Token ((11 3) on_sp " "))
 (Token ((11 4) on_ident "sum"))
 (Token ((11 7) on_lparen "("))
 (Token ((11 8) on_int "5"))
 (Token ((11 9) on_comma ","))
 (Token ((11 10) on_sp " "))
 (Token ((11 11) on_int "5"))
 (Token ((11 12) on_rparen ")"))
 (Token ((11 13) on_nl "\n"))
 (Token ((12 0) on_ident "u"))
 (Token ((12 1) on_sp " "))
 (Token ((12 2) on_op "="))
 (Token ((12 3) on_sp " "))
 (Token ((12 4) on_ident "sum"))
 (Token ((12 7) on_sp " "))
 (Token ((12 8) on_int "5"))
 (Token ((12 9) on_comma ","))
 (Token ((12 10) on_sp " "))
 (Token ((12 11) on_int "5"))
 (Token ((12 12) on_nl "\n")))
